CHANGELOG.md

```diff
@@ -5,6 +5,16 @@
 
 ## Swift 5.10
 
+* [SE-0412][]:
+
+  Under strict concurrency checking, every global or static variable must be either isolated to a global actor or be both immutable and of `Sendable` type.
```

Comment:
Can you show an example of some code that previously could exhibit data races at runtime that is now diagnosed under the proposal?